Source/Core/Core/HW/DVDThread.cpp

```diff
@@ -47,9 +47,9 @@ static u32 s_output_address;
 static u32 s_length;
 static bool s_decrypt;
 
-// Used to notify emulated software after executing command.
-// Pointers don't work with savestates, so CoreTiming events are used instead
-static int s_callback_event_type;
+// This determines which function will be used as a callback.
+// We can't have a function pointer here, because they can't be in savestates.
+static bool s_reply_to_ios;
```

Comment:
bools are used in savestates in other places too, like DVDInterface. If anything needs to be done about that before 5.0, it should be in a separate PR because it would touch more than the code that's related to this PR.